Misc/NEWS.d/next/Library/2022-04-10-20-10-59.bpo-46375.8j1ogZ.rst

```diff
@@ -0,0 +1 @@
+Add a ``peek()`` method to :class:`io.BytesIO`.
```

Comment:
```suggestion
Add :meth:`io.BytesIO.peek`.
```

Reply:
To render "peek()", you should write:

```
Add :meth:`~io.BytesIO.peek`
```

Reply:
I went with Erlend’s suggestion because it is more concise. I hope that’s ok.

Reply:
> To render "peek()", you should write:
> 
> ```
> Add :meth:`~io.BytesIO.peek`
> ```

See gh-109664